tests/test_plot.py

```diff
@@ -670,24 +632,27 @@ def test_scatter_plot_2d():
     assert (points.get_markeredgecolor() == 'C2')
     plt.close()
 
-    # Check q
+    # Check that q is ignored
     ax = plt.gca()
     scatter_plot_2d(ax, data_x, data_y, q=0)
     plt.close()
 
 
-@pytest.mark.parametrize('sigmas', [('1sigma', 0.682689492137086),
-                                    ('2sigma', 0.954499736103642),
-                                    ('3sigma', 0.997300203936740),
-                                    ('4sigma', 0.999936657516334),
-                                    ('5sigma', 0.999999426696856)])
+@pytest.mark.parametrize('sigmas', [(1, '1sigma', 0.682689492137086),
+                                    (2, '2sigma', 0.954499736103642),
+                                    (3, '3sigma', 0.997300203936740),
+                                    (4, '4sigma', 0.999936657516334),
+                                    (5, '5sigma', 0.999999426696856)])
 def test_quantile_plot_interval_str(sigmas):
-    q1, q2 = quantile_plot_interval(q=sigmas[0])
-    assert q1 == 0.5 - sigmas[1] / 2
-    assert q2 == 0.5 + sigmas[1] / 2
+    qi1, qi2 = quantile_plot_interval(q=sigmas[0])
+    qs1, qs2 = quantile_plot_interval(q=sigmas[1])
+    assert qi1 == pytest.approx(0.5 - sigmas[2] / 2)
+    assert qi2 == pytest.approx(0.5 + sigmas[2] / 2)
+    assert qs1 == pytest.approx(0.5 - sigmas[2] / 2)
+    assert qs2 == pytest.approx(0.5 + sigmas[2] / 2)
 
 
-@pytest.mark.parametrize('floats', [0, 1, 0.1, 0.9])
```

Comment:
The removal of `1` here because of its overlap with the new integer framework is a bit of a problem. How else could users specify 'please plot everything'? Perhaps a float (although that does also seem a bit dangerous)?

Reply:
The "please plot everything" is still covered by the `0`. So far `0` and `1` were synonymous, both returning `(0, 1)`...